### DISCUSSION TITLE
Very useful regex to use with Mantine UI, Chakra UI, or other Component Libraries

### DISCUSSION THREAD

**Discussion by maddada** [moalamri/vscode-inline-fold#139] [Show and tell]
Played around a bunch until I got this just right.
It won't collapse tags with under 20 characters coming after the tag name.

"inlineFold.regex": "(<[a-zA-Z]*\s)(.{20,})\/?>",
"inlineFold.regexFlags": "gm",
"inlineFold.regexGroup": 2,
"inlineFold.unfoldedOpacity": 1,
"inlineFold.unfoldOnLineSelect": true

![image](https://github.com/moalamri/vscode-inline-fold/assets/1892053/5b230e7a-6249-4ad8-a04a-7614d62cf751)

**Comment by moalamri** [comment]
Awesome :)
You are welcome to make a pr with this example in the README

**Comment by maddada** [comment]
@moalamri created the PR here: https://github.com/moalamri/vscode-inline-fold/pull/140

Thanks for the awesome extension!
Only wish it would support multiline tags (It's possible to collapse them now, but the regex is a bit complicated and it shows blank spaces between the tags until you click on the tag)

**Reply by moalamri** [reply]
True, it can't fold multi lines

> The extension can't fold multiline because it's basically a decorator not a real API (VS Code doesn't support inline-folding and it's been in the backlog for 4 years).
If you were to fold multiline with this extension then you will have empty lines due to being decorated to be hidden and not actually folded 😞.

Thanks for the PR
